deepset_cloud_sdk/_api/deepset_cloud_api.py

```diff
@@ -67,6 +68,12 @@ async def factory(cls, config: CommonConfig) -> AsyncGenerator[DeepsetCloudAPI,
         async with httpx.AsyncClient() as client:
             yield cls(config, client)
 
+    @retry(
+        retry=retry_if_exception_type((httpx.ReadTimeout, httpx.ReadError)),
```

Comment:
are there any other errors we want to retry on (even if we haven't encountered them yet)
https://www.python-httpx.org/exceptions/

Reply:
I added a retry for all `RequestError`'s

Reply:
done